src/Conversion/ONNXToKrnl/NN/Conv.cpp

```diff
@@ -12,6 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "src/Compiler/CompilerOptions.hpp"
```

Comment:
See Conversion/ONNXToKrnl/Math/Mathmul.cpp for example

```C++
  ONNXMatMulOpLowering(
      TypeConverter &typeConverter, MLIRContext *ctx, bool enableTiling)
      : ConversionPattern(
            typeConverter, mlir::ONNXMatMulOp::getOperationName(), 1, ctx),
        enableTiling(enableTiling) {}
  bool enableTiling;
...
```

We decided that we would rather not have the options directly used in the individual operations, but when we create the pass and add each pattern, we would specify there the optimizations desired.